app/models/game.rb

```diff
@@ -4,4 +4,32 @@ class Game < ApplicationRecord
   has_one :black_player, class_name: 'User', dependent: false
   has_one :winning_player, class_name: 'User', dependent: false
   has_many :pieces, dependent: false
+
+  def populate_board!
+    Rook.create(position_x: 0, position_y: 0, game: self, color: 'black', user_id: 1)
```

Comment:
user_id will need to be fixed later.

Either it can be `nil`. We don't need it (could use your new color column to know who owns what).

Or will need to be set to `black_player_id` or `white_player_id`